pkg/controller/gitopsconfig/controller.go

```diff
@@ -282,14 +284,14 @@ func (r *Reconciler) createCronJob(instance *gitopsv1alpha1.GitOpsConfig) error
 			update = false
 		} else {
 			// Error reading the object - requeue the request.
-			return err
+			return xerrors.Errorf("cannot read kubernetes object: %w", err)
```

Comment:
please try to add more context to this message; see also comment on l.174

Reply:
Done.